libs/quick-add/src/lib/quick-add-fab.component.scss

```
@@ -131,3 +131,30 @@
     opacity: 0.7;
   }
 }
+
+.goal-submenu {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  gap: 6px;
+  // Indented so the submenu reads as belonging to the goal button below
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)